mlflow/store/cli.py

```diff
@@ -0,0 +1,101 @@
+from mlflow.utils.logging_utils import eprint
+
+import click
+
+from mlflow.tracking import _get_store
+from mlflow.store.artifact_repo import ArtifactRepository
+from mlflow.utils.proto_json_utils import message_to_json
+
+
+@click.group("artifacts")
+def commands():
+    """Upload, list, and download artifacts from an MLflow artifact repository."""
+    pass
+
+
+@commands.command("log-artifact",
```

Comment:
I'd call this `log` and the other one `log-in-dir`. The names log_artifact and log_artifacts are confusing and I think we should rename the second one to log_artifacts_in_dir eventually too.